index.html

```diff
@@ -1,252 +1,248 @@
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="img/Vector.png" alt="twitter picture">
+                            </a>
                         </li>
                         <li>
-                            <div class="languages-skills">
-                                <img src="img/Frame 9 (1).png">
-                            </div>
+                            <a href="#">
+                                <img src="img/Vector1.png" alt="Gmail picture">
+                            </a>
                         </li>
                     </ul>
-                </section>
+                    <button class="btn-resume">Get my resume</button>
+                </div>
+                <div class="tool">
+                    <button class="language-button">Languages <span class="fa fa-chevron-down"></span></button>
+                    <ul class="languages">
+                        <li>
+                            <a href="#"><i class="fa-brands fa-html5"></i>html</a>
+                        </li>
+                        <li>
+                            <a href="#"><i class="fa-brands fa-css3-alt"></i>css</a>
+                        </li>
+                        <li>
+                            <a href="#"><i class="fa-brands fa-js"></i>javascript</a>
+                        </li>
+                    </ul>
+                    <button>frameworks <span class="fa fa-chevron-right"></span></button>
+                    <button>Skills <span class="fa fa-chevron-right"></span></button>
+                </div>
+            </div>
         </section>
 
-        <section class="contact-me">
-            <br />
-            <h6>Contact me</h6>
-            <p>
-                If you have an application you are interested in developing, a feature 
-                that you need built or a project that needs coding. I’d love to help 
-                with it. 
-            </p>
-            <form action="https://formspree.io/f/mrgjrqoe" method="POST">
-                <input type="test" name="name" placeholder="Full name" maxlength="30" required/>
-                <input type="email" name="email address" placeholder="Email address" required/>
-                <textarea name="contact" cols="30" rows="10" maxlength="500" placeholder="write you message here" required></textarea>
-                <div class="but">
-                    <button type="submit">Get in touch</button>
+        <footer>
+            <section class="contact">
+                <div class="contact-container">
+
+                    <h2>Contact me</h2>
+                    <p>
+                        If you have an application you are interested in developing, a feature
+                        that you need built or a project that needs coding. I’d love to help
+                        with it.
+                    </p>
```

Comment:
- Kindly make sure you aline This  paragraph as it shows in the Figma design, I can suggest adding some padding left and right to fix that :+1:  it will make your design much more to the Figma design as required in the project  :+1: 

- Kindly remove the border radius to your input fields it will make them look like the Figma design :+1:
- Kindly make also sure you add the background image in your contact page  as  shown in the Figma design  

## Your design 

![image](https://user-images.githubusercontent.com/95297251/221250320-4210a92b-e252-40c9-bf22-932308e9b984.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/221250132-dd576c20-bf0e-4a55-9ad6-59152f2b3dda.png)